src/http/server/response.cr

```diff
@@ -104,6 +104,17 @@ class HTTP::Server
       @output.close
     end
 
+    # Generates an error response using *message* and *code*.
+    #
+    # Calls `reset` and then writes the given message.
+    def respond_with_error(message = "Internal Server Error", code = 500)
+      reset
+      @status_code = code
+      @content_type = "text/plain"
```

Comment:
I think this should be `self.content_type = "text/plain"`, there's no `@content_type` variable (it's in the headers)